lib/redis/commands/lists.rb

```diff
@@ -99,18 +99,21 @@ def rpushx(key, value)
       #
       # @param [String] key
       # @param [Integer] count number of elements to remove
-      # @return [String, Array<String>] the values of the first elements
+      # @return [nil, String, Array<String>] the values of the first elements,
+      #     or nil when the key does not exist
```

Comment:
If the key doesn't exist, it's treated as an empty list.

I don't think we need to point that out particularly. Otherwise we'd have to say that on pretty much all commands documentation.

Reply:
NB: I'm only talking about the text change, the added `nil` type is fine by me.